lib.php

```diff
@@ -70,7 +70,7 @@ function local_honorlockproctoring_extend_navigation(global_navigation $navigati
             $pageurl = $PAGE->url->get_path();
 
             // Extension Check/Exam Instructions page.
-            if (str_contains($pageurl, "/mod/quiz/startattempt.php")) {
+            if (strpos($pageurl, "mod/quiz/startattempt.php")) {
```

Comment:
I imagine the removal of the initial `/` is because of the problem with strpos returning 0? If that is the case, woudn't it be better to do a strict !== false?

Reply:
`if (strpos($pageurl, "mod/quiz/startattempt.php") !== false) // will prevent position 0 from failing to determine it is not false`